modules/bullet/godot_ray_world_algorithm.cpp

```diff
@@ -102,7 +100,7 @@ void GodotRayWorldAlgorithm::processCollision(const btCollisionObjectWrapper *bo
 
 		btScalar depth(ray_shape->getScaledLength() * (btResult.m_closestHitFraction - 1));
 
-		if (depth >= -RAY_STABILITY_MARGIN)
+		if (depth >= -ray_shape->getMargin())
```

Comment:
This seems good, but you should remove the margin here:
/godot/modules/bullet/btRayShape.cpp:100

Reply:
That makes sense. I'll do so.